mod_reforged/hooks/config/strings.nut

```diff
@@ -1731,7 +1732,32 @@ foreach (vanillaDesc in vanillaDescriptions)
 		}]
 	}),
 	RF_RealizedPotential = ::UPD.getDescription({
-		Fluff = "From bones to brawn! This character has truly come a long way. Who was once a dreg of society is now a full-fledged mercenary. " + ::MSU.Text.colorPositive("\n\nAll perk points have been refunded and attributes increased."),
+		Fluff = "From bones to brawn! This character has truly come a long way. Who was once a dreg of society is now a full-fledged mercenary.",
+		Effects = [
+			{
+				Type = ::UPD.EffectType.Passive,
+				Description = [
+					"Require " + ::MSU.Text.colorNegative("100%") + " more daily wage.",
+				]
+			},
+			{
+				Type = ::UPD.EffectType.OneTimeEffect,
+				Description = [
+					"Refund all spent Perk Points.",
+					"Gain " + ::MSU.Text.colorPositive("+1") + " Perk Point.",
+					"Gain " + ::MSU.Text.colorPositive("1") + " new random Shared Perk Group.",
+					"Gain " + ::MSU.Text.colorPositive("2") + " new random Weapon Perk Groups.",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Melee Skill.|Concept.MeleeSkill]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Melee Defense.|Concept.MeleeDefense]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Ranged Skill.|Concept.RangedSkill]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Ranged Defense.|Concept.RangedDefense]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Hitpoints.|Concept.Hitpoints]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Maximum Fatigue.|Concept.Stamina]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Initiative.|Concept.Initiative]",
+					"Gain " + ::MSU.Text.colorPositive("+15") + " [Resolve.|Concept.Bravery]"
```

Comment:
All of these should be combined into a single bullet point: `All attributes are increased by +15`.